src/vishakha/ArrayEx1.java

```diff
@@ -0,0 +1,56 @@
+package vishakha;
+
+public class ArrayEx1 {
+	
+	int getArrayMax(int[] num){
+		int max = num[0];
+		for(int index=0; index<num.length; index++){
```

Comment:
Video sent on skype for this code, pls go through it

Also Check this code for reference
https://github.com/KrishnaKTechnocredits/JAVATechnoOct2020/pull/318/files